plugin-k8s/src/cgroup_v2.rs

```diff
@@ -40,27 +59,68 @@ pub fn is_accessible_dir(path: &Path) -> bool {
     path.is_dir()
 }
 
-
 /// Returns a Vector of CgroupV2MetricFile associated to pods availables under a given directory.
-fn list_metric_file_in_dir(root_directory_path: &Path) -> anyhow::Result<Vec<CgroupV2MetricFile>> {
+fn list_metric_file_in_dir(
+    root_directory_path: &Path,
+    hostname: String,
+    kubernetes_api_url: String,
+) -> anyhow::Result<Vec<CgroupV2MetricFile>> {
     let mut vec_file_metric: Vec<CgroupV2MetricFile> = Vec::new();
     let entries = fs::read_dir(root_directory_path)?;
+    // Let's create a runtime to await async function and fullfill hashmap
+    let rt = tokio::runtime::Builder::new_current_thread().enable_all().build()?;
```

Comment:
```suggestion
    // Let's create a runtime to await async function and fulfill hashmap
```

Reply:
I don't understand what you want me to change

Reply:
FULFILL